incubator/virtualcluster/pkg/vn-agent/server/route.go

```diff
@@ -91,6 +140,33 @@ func (s *Server) proxy(req *restful.Request, resp *restful.Response) {
 	handler.ServeHTTP(resp.ResponseWriter, req.Request)
 }
 
+func (s *Server) proxyStream(req *restful.Request, resp *restful.Response) {
```

Comment:
Is it possible to merge proxyStream and proxy method?  They are very similar except upgradeRequired flag. 
Will httpstream.IsUpgradeRequest(req.Request) always return false for non-stream APIs?

Reply:
good catch!